docs/management_api.yml

```diff
@@ -2034,6 +2058,30 @@ definitions:
     example:
       uri: http://mender.io/artifact.tar.gz.mender
       expire: 2016-10-29T10:45:34Z
+
+  ArtifactUploadLink:
+    description: URL for artifact file upload.
+    type: object
+    properties:
+      id:
+        type: string
+        format: uuid
+        description: The ID of the artifact upload intent.
+      uri:
+        type: string
+      expire:
+        type: string
+        format: date-time
+    required:
+      - id
+      - uri
+      - expire
```

Comment:
do we restrict somehow the expire? can I set expire to new geological era?

Reply:
The expiration time is a static configuration, you cannot set it using the API.